### ISSUE TITLE
NullPointerException when trying to start udp server

### ISSUE THREAD

**Issue by Dandraghas** [plasmoapp/plasmo-voice#352]
when i try to launch plasmo-voice on velocity server i get this:

```
[22:36:19 ERROR] [PlasmoVoice]: Failed to start the udp server
java.lang.NullPointerException: Cannot invoke "io.netty.channel.socket.nio.NioDatagramChannel.close()" because "this.channel" is null
        at su.plo.voice.proxy.socket.NettyUdpProxyServer.stop(NettyUdpProxyServer.java:66) ~[?:?]
        at su.plo.voice.proxy.socket.NettyUdpProxyServer.start(NettyUdpProxyServer.java:58) ~[?:?]
        at su.plo.voice.proxy.BaseVoiceProxy.startUdpServer(BaseVoiceProxy.java:236) ~[?:?]
        at su.plo.voice.proxy.BaseVoiceProxy.loadConfig(BaseVoiceProxy.java:181) ~[?:?]
        at su.plo.voice.proxy.BaseVoiceProxy.onInitialize(BaseVoiceProxy.java:102) ~[?:?]
        at su.plo.voice.velocity.VelocityVoiceProxy.onProxyInitialization(VelocityVoiceProxy.java:71) ~[?:?]
        at su.plo.voice.velocity.Lmbda$10.execute(Unknown Source) ~[?:?]
        at com.velocitypowered.proxy.event.UntargetedEventHandler$VoidHandler.lambda$buildHandler$0(UntargetedEventHandler.java:56) ~[velocity.jar:3.2.0-SNAPSHOT (git-bda1430d-b259)]
        at com.velocitypowered.proxy.event.VelocityEventManager.fire(VelocityEventManager.java:597) ~[velocity.jar:3.2.0-SNAPSHOT (git-bda1430d-b259)]
        at com.velocitypowered.proxy.event.VelocityEventManager.lambda$fire$5(VelocityEventManager.java:478) ~[velocity.jar:3.2.0-SNAPSHOT (git-bda1430d-b259)]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:833) [?:?]
```
plasmo-voice: 2.0.5
velocity: 3.2.0-SNAPSHOT 259

**Comment by HansAnonymous** [comment]
Can confirm this issue occurs on:
- PlasmoVoice-Velocity-2.0.6
- Velocity 3.2.0-SNAPSHOT 259
- Java 17.0.7+8-LTS-225

**Comment by Ethmangameon** [comment]
I am also having this issue on Velocity

- PlasmoVoice-Velocity-2.0.6
- Velocity 3.2.0-SNAPSHOT

**Comment by apehum** [comment]
NPE was fixed in 2.0.7: https://github.com/plasmoapp/plasmo-voice/releases/tag/2.0.7
